R/drive_publish.R

```diff
@@ -1,9 +1,15 @@
-#' Publish Google Drive file.
+#' Publish Google Drive file
```

Comment:
I *think* I left it to be explicit (that only the Google type files work), but perhaps that can just go in the description?